fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -228,7 +227,7 @@ protected RdfStream getResourceTriples() {
     protected RdfStream getResourceTriples(final int limit) {
         // use the thing described, not the description, for the subject of descriptive triples
         if (resource() instanceof NonRdfSourceDescription) {
```

Comment:
Is it still necessary to check if `resource()` is and `instanceof NonRdfSourceDescription`?